finalPrep/annotations.js

```diff
@@ -2,41 +2,41 @@
 // Use the double // to create a new comment
 
 //Build a Bear
-function buildABear(name, age, fur, clothes, specialPower) {
-  var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;
-  var demographics = [name, age];
-  var powerSaying = "Did you know that I can " + specialPower + " ?";
-  var builtBear = {
-    basicInfo: demographics,
-    clothes: clothes,
-    exterior: fur,
-    cost: 49.99,
-    sayings: [greeting, powerSaying, "Goodnight my friend!"],
-    isCuddly: true,
-  };
+function buildABear(name, age, fur, clothes, specialPower) { //Create a function called buildABear and give it 5 arguments: name, age, fur, clothes, specialPowers
+  var greeting = `Hey partner! My name is ${name} - will you be my friend?!`; //define greeting variable with string
+  var demographics = [name, age]; //define demographics variable with name and age
+  var powerSaying = "Did you know that I can " + specialPower + " ?"; //define powerSaying variable with string concatinated with specialPower variable
+  var builtBear = { //declare a new varialble builtBear and assign it to an object with these key/value pairs:
+    basicInfo: demographics, //basicInfo key with demographics variable as value
+    clothes: clothes, //clothes key with clothes variable as value
+    exterior: fur, //exterior key with fur variable as value
```

Comment:
You can think about distinguishing what values were declared as variables, from ones that have been passed as parameters.